runner/local/simple_test.go

```diff
@@ -112,6 +114,10 @@ func aborted() runner.ProcessStatus {
 	return runner.AbortStatus(runner.RunId(""))
 }
 
+func badRequest(errorText string) runner.ProcessStatus {
+	return runner.BadRequestStatus(runner.RunId(""), fmt.Errorf(errorText))
+}
+
 func assertRun(t *testing.T, r runner.Runner, expected runner.ProcessStatus, args ...string) runner.RunId {
```

Comment:
Let's move this conversation to our Testing Best Practices page (https://confluence.twitter.biz/display/EE/Testing+Best+Practices)